Mathlib/Algebra/Category/Ring/Instances.lean

```diff
@@ -36,16 +37,51 @@ instance Localization.epi' {R : CommRingCat} (M : Submonoid R) :
   rcases R with ⟨α, str⟩
   exact IsLocalization.epi M _
 
-instance CommRingCat.isLocalRingHom_comp {R S T : CommRingCat} (f : R ⟶ S) (g : S ⟶ T)
-    [IsLocalRingHom g] [IsLocalRingHom f] : IsLocalRingHom (f ≫ g) :=
-  RingHom.isLocalRingHom_comp _ _
+-- These three instances solves the issue where the `FunLike` instances provided by
+-- `CommRingCat.instFunLike'`, `CommRingCat.instFunLike''`, and `CommRingCat.instFunLike'''`
+-- are not syntactically equal to `CommRingCat.instFunLike` when applied to
+-- objects of the form `CommRingCat.of R`.
+-- To prevent infinite loops, the priority of these three instances must be set lower
+-- than that of other instances.
```

Comment:
```suggestion
-- These three instances solve the issue where the `FunLike` instances provided by
-- `CommRingCat.instFunLike'`, `CommRingCat.instFunLike''`, and `CommRingCat.instFunLike'''`
-- are not syntactically equal to `CommRingCat.instFunLike` when applied to
-- objects of the form `CommRingCat.of R`.
-- To prevent infinite loops, the priority of these three instances must be set lower
-- than that of other instances.
```
Also can you use block comment syntax `/- ... -/` instead?

Reply:
Fixed. Thanks.